### PR TITLE
3.5 Manual peering spec

### PR DESCRIPTION
**Pull request by jkrvivian**

### CODE REVIEW

File: 3.5 Manual Peering.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+* Chosen neighbors (outbound): If `node < peer`, the node proactively sends a peering request to the peer.
+* Accepted neighbors (inbound): If `node > peer`, the node awaits the peer to send a peering request and accept it.
+
+After the group of peers is decided, the node shall either send peering requests to the chosen neighbors or wait for peering requests from accepted neighbors. The connections are checked every `reconnectInterval`, if a connection is dropped, the node shall try to re-establish it until the peer is removed or the node shutdowns.
+
+A node removes a peer by closing the connection directly without sending a drop message described in [Section 3.4 - Peer Selection](./3.4%20Peer%20Selection.md) to the peer. Thus, the removed peer shall still try to reconnect to the node every `reconnectInterval` up to its node operator removes the node from the manual peering list. That is, it is the node operators' responsibility to monitor and manage the known peer list in manual peering.
```
Comment (darcy-camargo) [inline 3.5 Manual Peering.md:14]:
```suggestion
A node removes a peer by closing the connection directly without sending a drop message described in [Section 3.4 - Peer Selection](./3.4%20Peer%20Selection.md) to the peer. Thus, the removed peer shall still try to reconnect to the node every `reconnectInterval`, until its operator removes the node from the manual peering list. That is, it is the node operators' responsibility to monitor and manage the known peer list in manual peering.
```

**Review by capossele — APPROVED** [review]

**Review by darcy-camargo — CHANGES_REQUESTED** [review]
What is her is really good, I would add some lines about the risks and care on using Manual Peering instead of Autopeering (since Autopeering is not mandatory it seems)

File: 3.5 Manual Peering.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# 3.5 Manual Peering
+Manual peering enables node operators to manually manage nodes' neighbors, orthogonal to the peering protocols described in [Section 3.3 - Peer Discovery](./3.3%20Peer%20Discovery.md) and [Section 3.4 - Peer Selection](./3.4%20Peer%20Selection.md). This section defines manual peering protocol.
+
+## 3.5.1 Motivation
+Manual peering can be additional protection against eclipse attacks as the manual peering is completely in the hands of the node operator based on real-world trust. Furthermore, it allows operating nodes without exposing their IP addresses to the network.
```
Comment (oliviasaa) [inline 3.5 Manual Peering.md:5]:
```suggestion
Manual peering can be an additional protection against eclipse attacks as the manual peering is completely in the hands of the node operator based on real-world trust. Furthermore, it allows operating nodes without exposing their IP addresses to the network.
```

**Review by oliviasaa — APPROVED** [review]
I left a suggestion in the 1st paragraph, but it's fine anyway :)

**Review by darcy-camargo — APPROVED** [review]

**Pull request merged**